compute_endpoint/tests/unit/test_endpoint_unit.py

```diff
@@ -92,7 +91,7 @@ def mock_ep_data(fs):
 def mock_ep_buf():
     buf = io.StringIO()
     # Endpoint.get_endpoint
-    # ep = mocker.patch("funcx_endpoint.endpoint.endpoint.Endpoint.get_endpoints")
+    # ep = mocker.patch("globus_compute_endpoint.endpoint.endpoint.Endpoint.get_endpoints")  # noqa: E501
```

Comment:
Might as well just remove this line; looks like unintentionally left over from a refactor.  (Maybe to using the `_mock_base` concept?)

Reply:
Removed